proto/vault/v1/vault.proto

```diff
@@ -18,4 +18,17 @@ message VaultAccount {
   repeated string underlying_assets = 3;
   // admin is the address that has administrative privileges over the vault.
   string admin = 4 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // interest_rate is the annualized continuous interest rate for the vault,
+  // represented as a decimal string (e.g., "0.05" for 5% APR).
+  string interest_rate = 5 [(cosmos_proto.scalar) = "cosmos.DecString"];
```

Comment:
We might want to have a "desired rate" and a "current rate" tracked.  That way we can easily show if there is a difference and the interest has been temporarily overridden (such as from lack of funds)

Reply:
So, if I understand it right.  This is what we would use to disable the vault when funds are no longer there?  For example, for a running vault the rate would be 3.8 for both.  Then current rate would be changed to 0 when the reserve funds are not enough.  If so, I can add this update to our #9 issue.

Reply:
Disable the interest vs disable the whole vault… but yes the rest seems right. I don’t know if there is any other reason these wouldn’t match up though.